CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 ### Fixed issues
 - Changed driver behavior to allow prepared statement objects to be reused, preventing a "multiple queries are not allowed" error [#2482](https://github.com/microsoft/mssql-jdbc/pull/2482)
 - Adjusted DESTINATION_COL_METADATA_LOCK, in SQLServerBulkCopy, so that is properly released in all cases [#2484](https://github.com/microsoft/mssql-jdbc/pull/2484)
+- Fixed connection retry behavior when `connectRetryCount` is set to a value greater than 1 [#2513](https://github.com.mcas.ms/microsoft/mssql-jdbc/pull/2513)
```

Comment:
fix url

Reply:
this is missing some PRs see all PRs tagged for milestone https://github.com/microsoft/mssql-jdbc/pulls?q=is%3Apr+milestone%3A12.9.0+

Reply:
Odd, my URL for these PRs are different than public facing ones. Fixed.